Data-Structures/HashTable/HashTable.js

```diff
@@ -0,0 +1,105 @@
+export default class HashTable {
+  constructor(limit = 100) {
+    // Initialize the storage and limit variables
+    this.storage = new Array(limit)
```

Comment:
It would be helpful to call this property `this.#buckets` (it should also be private).